### PR TITLE
Criteo Adapter: remove bid.adId

### PR DESCRIPTION
**Pull request by dzhang-criteo**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change

Removed bid.adId parameter as asked (see #6381)

### CODE REVIEW

**Comment by patmmccann** [comment]
please note the introduced lgtm alert

**Review by patmmccann — CHANGES_REQUESTED** [review]
please fix the lgtm alert, tyvm! 

Also, you should do a docs pull to remove the warning.

File: modules/criteoBidAdapter.js [hunk 1]
```diff
@@ -235,7 +235,6 @@ export const spec = {
         const bidId = bidRequest.bidId;
         const bid = {
           requestId: bidId,
-          adId: slot.bidId || getUniqueIdentifierStr(),
```
Comment (patmmccann) [inline modules/criteoBidAdapter.js:238]: please remove imprt of getUniqueIdentifierStr

**Comment by dzhang-criteo** [comment]
I have done the update.
I'll prepare a pull-request for the doc as you suggested.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**